### ISSUE TITLE
Change Static Injector's request/response after being created

### ISSUE THREAD

**Issue by Hestehviskeren** [MegatronKing/HttpCanary#21]
I am unable to change the request/response data after the Static Injector has been created - is this intended? as this makes the static injector very hard to use as a debug tool.

**Comment by MegatronKing** [comment]
The static injector should be working, do you have more details about this issue?

**Comment by Hestehviskeren** [comment]
> The static injector should be working, do you have more details about this issue?

The Static Injector work, correct - what I am experiencing is that once an static injector is created, it's data cannot be edited once the static injector is created (there is no edit function for the static injector).

**Comment by MegatronKing** [comment]
You are right, the injector doesn't support to be edited again, I will add this feature in future version.
